File: master/dataprocessing.py
import csv
import matplotlib.pyplot as plt
import numpy as np

maxes = []
means = []
first = True
with open("maxmean.csv","r") as f:
    rd = csv.reader(f)
    for row in rd:
        maxes.append(float(row[0]))
        means.append(float(row[1]))

plt.rc('font', family='serif')

plt.plot(maxes,label='$g_{max}$')
plt.ylim([975, 1005])
plt.legend(loc=4, prop={'size': 16})
plt.xlabel(r'pokolenie',fontsize=16)
plt.ylabel(r'$g(x_1, x_2, x_3, x_4)$',fontsize=16)
plt.savefig('levy_maxes')

plt.figure()
plt.plot(means,'-.',label='$g_{avg}$')
plt.legend(prop={'size': 16})
plt.xlabel(r'pokolenie',fontsize=16)
plt.ylabel(r'$g(x_1, x_2, x_3, x_4)$',fontsize=16)
plt.savefig('levy_means')

File: data_workspace/1_1/dataprocessing.m
clear

a = csvread('trudata.csv');
a(1,:) = [];
vals = a(:,3);
v = reshape(vals,100,max(a(:,1)));

maxes = max(v);
means = mean(v);

plot(maxes,'linewidth',2)
legend('\fontsize{20} g_{max}')
xlabel('pokolenie',"fontsize", 20)
ylabel('g(x_1, x_2, x_3, x_4)',"fontsize", 20)
figure(2)
plot(means,'-.','linewidth',2)
legend('\fontsize{20} g_{avg}')
xlabel('pokolenie',"fontsize", 20)
ylabel('g(x_1, x_2, x_3, x_4)',"fontsize", 20)

csvwrite('maxmean.csv',[maxes', means'])